src/psyclone/transformations.py

```diff
@@ -2548,6 +2554,10 @@ def validate(self, nodes, options):
                 "A kernels transformation must enclose at least one loop or "
                 "array range but none were found.")
 
+        # do not has mixed async
+        if options == None:
+            options = {}
+        async_queue = options.get('async_queue', False)
 
```

Comment:
We need to check that this is a valid value?

Reply:
Also, please could you clarify what the comment at L2579 means.

Reply:
Hum damn, with the changes I don't see about what comment you are speeking here ?

Reply:
I should have stuck with the original line number - I thought I was being helpful by using the latest value! It's the one at 2557 in the snippet that github is showing above.